.github/ISSUE_TEMPLATE/BUG_BACKEND.md

```diff
@@ -0,0 +1,35 @@
+---
+name: ⚠️ Report Backend Bug
+about: Report a Backend bug on Kubeflow Pipelines
+title: "[backend] <Bug Name>"
+labels: kind/bug, area/backend
+---
+
+### Environment
+
+*  How did you deploy Kubeflow Pipelines (KFP)?
+<!-- See [an introduction of all options](https://www.kubeflow.org/docs/pipelines/installation/overview/). -->
+*  KFP version: <!-- See version number shows on bottom of KFP UI left sidenav. -->
+*  KFP SDK version: <!-- Please attach the output of this shell command: $pip list | grep kfp -->
+
+
+### Steps to reproduce
+
+<!--
+How can we make the problem occur?
+This could be a description, code snippet, log output, screenshot, etc.
+-->
+
+### Expected result
+
+<!-- What should the correct behavior be? -->
+
+### Materials and Reference
+
+<!-- Help oncaller to debug by providing sample code, background context, links to reference, etc. -->
```

Comment:
Suggested revision, please replace all instances:

Help us debug this issue by providing resources such as: sample code, background context, or links to references.

Reply:
Sounds good! Applied.